src/app/nation/nation.component.html

```diff
@@ -14,11 +15,13 @@ <h1 i18n>Nation List</h1>
         </tr>
       </thead>
       <tbody>
-        <tr *ngFor="let nation of nations">
+        <ng-container *ngFor="let nation of nations; let i = index">
+        <tr *ngIf="nation.doc.name.toLowerCase().includes(searchText)">
```

Comment:
@razu9861 "nation.doc.name.toLowerCase()" why nation name is in lowercase we can filter as if it is capital . Only letter must be in correct form.